tracer/src/Datadog.Tracer.Native/tracer_tokens.cpp

```diff
@@ -144,6 +144,49 @@ const shared::WSTRING& TracerTokens::GetCallTargetReturnGenericType()
     return managed_profiler_calltarget_returntype_generics;
 }
 
+HRESULT TracerTokens::EnsureBaseCalltargetTokens()
+{
+    HRESULT hr = CallTargetTokens::EnsureBaseCalltargetTokens();
+    if (FAILED(hr))
+    {
+        return hr;
+    }
+    // *** Ensure Datadog.Trace.ClrProfiler.CallTarget.CallTargetBubbleUpException type ref, might not be available if tracer version is < 2.22
+    if (bubbleUpExceptionTypeRef == mdTypeRefNil)
```

Comment:
Were you able to the version mismatch scenario? I'd like to make sure that's tested before merging the changes

Reply:
yes, tests are passing now. I dont test on the version though but on the [availability of the type](https://github.com/DataDog/dd-trace-dotnet/blob/b9b2c8c95e0dc5c966184933b404f5d1ed7d06ba/tracer/src/Datadog.Tracer.Native/cor_profiler.cpp#L2528)
indeed, testing on the version didnt work well for me, as some tests use a 2.255 dummy version

Reply:
@zacharycmontoya Just as a clarification, `CallTargetBubbleUpException` typeref is in use only if `corProfiler->call_target_bubble_up_exception_available` yields `true`, which happens when it's available ([reference](https://github.com/DataDog/dd-trace-dotnet/blob/b9b2c8c95e0dc5c966184933b404f5d1ed7d06ba/tracer/src/Datadog.Tracer.Native/cor_profiler.cpp#L2528)).
Kudos @anna-git on handling this scenario.